cli/cmd/cluster.go

```diff
@@ -217,3 +235,48 @@ func promptForEmail() {
 		}()
 	}
 }
+
+func refreshCachedClusterConfig(awsCreds *AWSCredentials) *clusterconfig.ClusterConfig {
+	userClusterConfig := &clusterconfig.ClusterConfig{}
+	err := clusterconfig.SetFileDefaults(userClusterConfig)
+	if err != nil {
+		errors.Exit(err)
+	}
+	if flagClusterConfig != "" {
+		err := readClusterConfigFile(userClusterConfig, flagClusterConfig)
+		if err != nil {
+			errors.Exit(err)
+		}
+	}
+
+	cachedClusterConfig := &clusterconfig.ClusterConfig{}
+	readClusterConfigFile(cachedClusterConfig, cachedClusterConfigPath)
+
+	if len(userClusterConfig.ClusterName) == 0 {
+		userClusterConfig.ClusterName = cachedClusterConfig.ClusterName
+	}
+
+	if userClusterConfig.Region == nil {
+		userClusterConfig.Region = cachedClusterConfig.Region
+	}
+
+	if len(userClusterConfig.ClusterName) == 0 {
+		errors.Exit("unable to find cluster_name; please configure `cluster_name` to the name of an existing Cortex cluster or create a Cortex cluster with `cortex cluster up`")
+	}
+
+	if userClusterConfig.Region == nil {
+		errors.Exit("unable to find region; please configure `region` to the S3 region of an existing Cortex cluster or create a Cortex cluster with `cortex cluster up`")
```

Comment:
Do we have a variable for `"region"`? If not we don't necessarily need to add one, but if we do, we might as well use it